tests/infra/remote.py

```diff
@@ -133,16 +134,15 @@ def __init__(
         stop()  disconnects, which shuts down the command via SIGHUP
         """
         self.hostname = hostname
-        # For SSHRemote, both executable files (host and enclave) and data
-        # files (ledger, secrets) are copied to the remote
-        self.files = exe_files
-        self.files += data_files
+        self.exe_files = exe_files
```

Comment:
That seemed like a useful comment?

Reply:
True. I've moved it to `SSHRemote`'s `_setup_files()` though.